### DISCUSSION TITLE
Banned characters in reference yaml configuration file

### DISCUSSION THREAD

**Discussion by Dego1n** [golangci/golangci-lint#3957] [Q&A] [answered]
Is there any reason why character "7" is banned in reference file (.golangci.reference.yml) ?

It was added in https://github.com/golangci/golangci-lint/pull/2520

This does not make any sense.

**Comment by ldez** [comment, marked as answer]
> ```
> # This file contains all available configuration options
> # with their default values (in comments).
> #
> # This file is not a configuration example,
> # it contains the exhaustive configuration with explanations of the options.
> ```

https://github.com/golangci/golangci-lint/blob/d58f1401710f56d06e2a54e460f160f72b8499d2/.golangci.reference.yml#L1C1-L5C77
